jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/html/RangeInputLoader.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.flowui.xml.layout.loader.html;
+
+import com.vaadin.flow.component.html.RangeInput;
+import io.jmix.flowui.xml.layout.loader.AbstractComponentLoader;
+
+public class RangeInputLoader extends AbstractComponentLoader<RangeInput> {
+    @Override
+    protected RangeInput createComponent() {
+        return factory.create(RangeInput.class);
+    }
+
+    @Override
+    public void loadComponent() {
+        componentLoader().loadEnabled(resultComponent, element);
```

Comment:
You load only direct attributes of the component and don't load inherited attributes since `AbstractComponentLoader` loads nothing. Please check component API and load everything